Introduction_and_Goals.md

```diff
@@ -36,16 +36,16 @@ See [WikibaseClient editor](./systems/WikibaseClient/01-Introduction_and_Goals.m
 
 ### Researcher
 
-- Quickly create visualizations based on data from Wikidata without the need to download it.
+See [Researcher](./systems/WikibaseRepo/01-Introduction_and_Goals.md#researcher)
 
 ### Non Wikimedia Wikibase Project
 
 - Create knowledge bases similar to Wikidata using our same software stack (including extensions, tools and features)
 - Install, configure, update, and maintain a set of Wikibase systems with confidence.
 
-#### Large Non Wikimedia Wikibase Project
+### Third-party Tool Builder
```

Comment:
Why "Third-party"? Do you mean to imply it's volunteers who build the tools?

Reply:
Ok, _Tool Builder_ it is